x-pack/plugins/canvas/public/state/middleware/aeroelastic.js

```diff
@@ -206,7 +206,7 @@ export const aeroelastic = ({ dispatch, getState }) => {
 
     // set the selected element on the global store, if one element is selected
     const selectedShape = nextScene.selectedPrimaryShapes[0];
-    if (nextScene.selectedShapes.length === 1 && !isGroupId(selectedShape)) {
+    if (nextScene.selectedPrimaryShapes.length === 1 && !isGroupId(selectedShape)) {
```

Comment:
This is not strictly necessary, because the count of selected primary shapes (=elements) corresponds to the number of selected shapes (which can be either an element, or one of its control points) but as the non-primary elements are internal to the Layout Engine (except of course their rendered artifacts), only the primary shapes (ie. elements) should be used. With the TypeScript-initiated refactoring we'll likely hide what's currently called `selectedShapes`.